recipe/post-link.bat

```diff
@@ -0,0 +1,3 @@
+@echo off
+
+"%PREFIX%\Scripts\jupyter-nbextension.exe" enable jupyter_server_proxy --py --sys-prefix > "%PREFIX%\.messages.txt" 2>&1 && if errorlevel 1 exit 1
```

Comment:
If this is like `nbserverproxy`, we probably want to change this to `jupyter-serverextension`. Same for the other commands below.